peachjam/migrations/0132_backfill_outcomes.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 3.2.21 on 2024-05-15 13:59
+
+from django.db import migrations
+
+
+def forwards(apps, schema_editor):
+    Outcome = apps.get_model("peachjam", "Outcome")
+    OrderOutCome = apps.get_model("peachjam", "OrderOutCome")
+    Judgment = apps.get_model("peachjam", "Judgment")
+
+    for outcome in OrderOutCome.objects.all():
+        o = Outcome()
+        o.name = outcome.name
+        o.description = outcome.description
+        if hasattr(outcome, "name_en"):
+            o.name_en = outcome.name_en
+        if hasattr(outcome, "name_fr"):
+            o.name_fr = outcome.name_fr
+        if hasattr(outcome, "name_pt"):
+            o.name_pt = outcome.name_pt
+        if hasattr(outcome, "name_sw"):
+            o.name_sw = outcome.name_sw
+        o.save()
+
+    for judgment in Judgment.objects.all().iterator(chunk_size=100):
```

Comment:
```suggestion
    for judgment in Judgment.objects.filter(order_outcomes__exist=True).order_by('pk').distinct('pk').iterator(chunk_size=100):
```

this excludes anything that doesn't have an order_outcome